app/src/main/java/com/example/parkingsystem/mvp/contract/ParkingContract.kt

```diff
@@ -0,0 +1,15 @@
+package com.example.parkingsystem.mvp.contract
+
+interface ParkingContract {
+    interface Model {
+        fun getParkingSize(): Int
+    }
+
+    interface Presenter {
+        fun onParkingSizeButtonPressed()
+    }
+
+    interface View {
+        fun showParkingSizeToast(size: Int)
+    }
+}
```

Comment:
to avoid this error add an empty line at the end of the file

Reply:
Done 👍